spec/test_helper.rb

```diff
@@ -0,0 +1,29 @@
+def create_US_summaries
```

Comment:
Let's see if we can set up the helper using the RSpec conventions with making a helper so we don't have to explicitly include test_helper